packages/bpk-component-journey-arrow/src/BpkJourneyArrow.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { cssModules } from '../../bpk-react-utils';
+
+import STYLES from './BpkJourneyArrow.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+export type Props = {
+  stopClassName?: string,
+  /** the number of dot stops to display */
+  stops?: number,
+  [rest: string]: any; // Inexact rest. See decisions/inexact-rest.md
+
+};
+const BpkJourneyArrow = ({
+   stopClassName,
```

Comment:
What would we expect to come through in this property? Is this how we get around `className`?

Reply:
Thanks for the response. Given that our designs only show the current implementation as the only possiblity, I would say let's remove this as design will not want this opened up to be custom and devaite from the current designs